README.md

```diff
@@ -29,7 +29,7 @@ Check out the website: https://linkfree.eddiehub.io
 
 ## 👇 Prerequisites
 
-Before installation, please make sure you have already installed the following tools:
+Before Contributing or adding new feature, please make sure you have already installed the following tools:
```

Comment:
There's a missing `a` here.
```suggestion
Before contributing or adding a new feature, please make sure you have already installed the following tools:
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable